src/frontend/Komponenter/Personoversikt/BehandlingsoversiktTabell.tsx

```diff
@@ -101,6 +107,18 @@ export const BehandlingsoversiktTabell: React.FC<{
             ? behandlingOgTilbakekrevingsårsakTilTekst[behandling.årsak]
             : '-';
 
+    const utledKolonneForVedtaksdato = (
+        behandlingsresultat?: BehandlingResultat | TilbakekrevingBehandlingsresultatstype,
+        vedtaksdato?: string
+    ) => {
+        if (behandlingsresultat === BehandlingResultat.HENLAGT) {
```

Comment:
Husk at du ikke trenger denne hvis du legger til tidspunkt for henlagt

Reply:
Fikset i [64c0e3e](https://github.com/navikt/familie-ef-sak-frontend/pull/1164/commits/64c0e3e9c7b53c003e6abdd299bae061757bb859)